galaxy/values.yaml

```diff
@@ -267,6 +267,53 @@ extraEnv: []
   # - name: EXAMPLE_ENV
   #   value: MY_VALUE
 
+#- User defined CronJobs
+cronJobs:
+  chown:
+    schedule: "0 * * * *"
+    securityContext:
+      runAsUser: 0
+    command:
+      - /usr/bin/chown
+    args:
+      - "-R"
+      - "galaxy"
+      - "/galaxy/server/database"
```

Comment:
Why is the chown cronjob there? It doesn't seem right to me that it should be necessary.

Reply:
Yeah, sorry.  I am in the process of reverting that right now....